source/_components/doods.markdown

```diff
@@ -0,0 +1,160 @@
+---
+title: "DOODS"
+description: "Detect and recognize objects with DOODS."
+logo: home-assistant.png
```

Comment:
The Home Assistant logo can only be used for own integrations, so please remove the line if there is no logo.